charts/noe/templates/webhook.yaml

```diff
@@ -63,4 +68,9 @@ spec:
         - name: docker-config-{{ $i }}
```

Comment:
Won't execution fail if the file does not exist? If that's the case maybe we need a switch to define where are the container runtime configuration files and their format, be it docker or contained or another runtime.

Reply:
Yes it will fail.

But why you would filled it up the containerdConfigPathCandidates parameter if you are not using containerd?
The switch is the same parameter, why would you want to add another flag parameter?

Reply:
I wouldn't add another parameter, I was considering to use a single list of locations and wondering if an extra parameter for the type would be necessary. Nonetheless, it's something that can be refactored or changed later on, specially if other runtimes require other setups.